packages/chat-component/src/core/parser/index.ts

```diff
@@ -19,9 +19,16 @@ export async function parseStreamedMessages({
   let isFollowupQuestion = false;
   let stepIndex = 0;
   let textBlockIndex = 0;
+  const thoughtProcess: string[] = [];
 
   for await (const chunk of chunks) {
+    // eslint-disable-next-line no-prototype-builtins
```

Comment:
we should probably disable this rule, makes no sense to me

Reply:
Agreed.